### PR TITLE
chore: add publicFacet and instance to MakeInstanceResult

### PR DESCRIPTION
**Pull request by katelynsills**
Add the `publicFacet` and `instance` to MakeInstanceResult.

Needed for dapp-encouragement work.

### CODE REVIEW

**Review by Chris-Hibbert — APPROVED** [review]

File: packages/zoe/src/types.js [hunk 1]
```diff
@@ -106,14 +106,11 @@
...
-/**
- * @typedef {Object} CreatorFacetWInstance
- * @property {() => Instance} getInstance
- */
-
 /**
  * @typedef {Object} MakeInstanceResult
- * @property {CreatorFacetWInstance & Record<string, Function>} creatorFacet
+ * @property {Record<string, Function>} creatorFacet
+ * @property {{}} Instance
```
Comment (michaelfig) [inline packages/zoe/src/types.js:112]:
```suggestion
 * @property {Instance} instance
```

**Review by michaelfig — CHANGES_REQUESTED** [review]
Just a typing change.

File: packages/zoe/src/types.js (hunk 1 above)
Comment (katelynsills) [reply packages/zoe/src/types.js:112]: Op, yes, this is much better, thanks!

Comment (katelynsills) [reply packages/zoe/src/types.js:112]: Fixed on zoe-release-candidate

**Pull request merged**